awx/plugins/inventory/foreman.py

```diff
@@ -424,6 +439,8 @@ def _print_data(self):
                     'foreman': self.cache[hostname],
                     'foreman_params': self.params[hostname],
                 }
+                if self.want_ansible_ssh_host and 'ip' in self.cache[hostname]:
+                    self.inventory['_meta']['hostvars'][hostname]['ansible_ssh_host'] = self.cache[hostname]['ip']
```

Comment:
Since the default of `want_ansible_ssh_host` is false, I expect this to have no effect, so I'm not opposed to changing it.